sbe-tool/src/main/java/uk/co/real_logic/sbe/generation/rust/RustGenerator.java

```diff
@@ -117,7 +117,7 @@ public void generate() throws IOException
 
             indent(writer, 0, "[package]\n");
             indent(writer, 0, "name = \"%s\"\n", namespace);
-            indent(writer, 0, "version = \"0.1.0\"\n");
+            indent(writer, 0, "version = \"%s\"\n", ir.semanticVersion());
```

Comment:
This would be great, I +1 this.

Even better would be if we put this in codegen too, so you could start checking for semver against clients?

Reply:
I believe [using a semanticVersion is optional](https://github.com/real-logic/simple-binary-encoding/blob/a5e12d1c0d22e477ca302ee58c352984540e882f/sbe-tool/src/main/resources/fpl/sbe.xsd#L54-L60) and `ir.semanticVersion()` might be `null`.